exadata-database-service-on-cc/create-database/create-database.md

```diff
@@ -0,0 +1,104 @@
+
+        * Click **Create a new Database Home:**
+
+          * **Enter Database Home display name.**
+
+          * Click **Change Database Image** to select your software version. Select a Database Software Image window is displayed.
+
+          * Select an **Image Type**, **Oracle Provided Database Software Images**, or **Custom Database Software Images**. If you choose **Oracle Provided Database Software Images**, then you can use the **Display all available version** switch to choose from all available PSUs and RUs. The most recent release for each major version is indicated with a **latest** label.
+
+     * **Provide the name of the first PDB:** Specify the name for the first PDB. A PDB is created with the database.
+
+     * **Provide the administration password:** Provide and confirm the Oracle Database administration password.
+
+        * **Use the administrator password for the TDE wallet:** When this option is checked, the password entered for the SYS user is also used for the TDE wallet. To set the TDE wallet password manually, uncheck this option and enter the TDE wallet password.
+
+     * **Choose the database workload type:** Select the workload type that best suits your application from one of the following options:
+
+        * **Transactional Processing:** Select this option to configure the database for a transactional workload.
+
+        * **Data Warehouse:** Select this option to configure the database for decision support or data warehouse workload.
+
+     * **Backup Destination Type:** Select a backup destination for the database. Select the **Enable automatic backups:** to enable daily backups using the policy for automatic backups.
+
+        * **None:** Select to not define a backup configuration for the database.
+
+        * **Local:** Select to store backups locally in the Oracle Exadata Storage Servers on your Oracle Exadata Database Service on Cloud@Customer.
+
+        * **Object Storage:** Select to store backups in an Oracle-managed object storage container on Oracle Cloud Infrastructure.
+
+        * **NFS:** Select to store backups in one of your previously defined backup destinations that use Network File System (NFS) storage.
+
+        * **Recovery Appliance:** Select to store backups in one of your previously defined backup destinations that use Oracle Zero Data Loss Recovery Appliance.
+
+      * Select **Show Advanced Options.**
+
+        * **Provide the Oracle SID prefix:**
+
+        * **Backup retention period:** From the list, you can choose the length of time that you want automatic backups to be retained.
+
+        * **Character set:** The character set for the database.
+
+        * **National character set:** The national character set for the database.
+
+        * **Tags:** You can choose to apply tags.
+
+6. Click **Create Database.**
+
+
+
+## Learn More?
```

Comment:
@leoneila - Please remove the ? after Learn more

Reply:
removed the '?' from Learn more